### PR TITLE
Initial version of dxw cyber confidentiality sectiion

### PR DESCRIPTION
**Pull request by harry-m**
This adds a new openness and confidentiality section for dxw cyber, and clarifies for dxw digital staff that cyber things are private.

### CODE REVIEW

File: playbook.md [hunk 1]
```diff
@@ -77,9 +78,25 @@ However, there are some things that we must keep private.
...
+
+As a result, we take the opposite approach to dxw digital. We treat everything about our work as private and confidential, with these exceptions:
+
+- General discussions about tools, techniques, approaches and methodologies
+- The nature of our roles and day-to-day work
+- The purpose, mission, vision and values for dxw cyber, and the ways we get things done
+
+However, even with these exceptions, there is an overriding principle that client information must be kept confidential. For example, if talking about a particular attack technique could allow someone to infer the identity of a client, we would not discuss it in public.
+
+In addition, we take measures to ensure that we don't accidentally expose confidential information. For more information about this, talk to your team lead.
```
Comment (staceydxw) [inline playbook.md]: Who is the 'team lead' here? Might be worth being more explicit, is it a head of? Or a delivery manager?

File: playbook.md [hunk 2]
```diff
@@ -65,8 +65,9 @@ When taking time off during client work, we discuss how it will impact the sched
 
 Sending communication outside of working hours may create an unintended sense of urgency with the recipients of your message. Try to avoid creating that urgency when possible. Unless actually urgent, you may ignore messages which you receive outside of working hours and handle them once you are back at work.
 
-#### Openness
-We believe it's best to be open about what we're doing. We encourage our clients to do the same, as does the [service manual](https://www.gov.uk/service-manual). We talk, blog and write about what we're doing, and we are open about as much as we can.
+#### Openness and confidentiality (dxw digital)
+
+dxw digital believe it's best to be open about what we're doing. We encourage our clients to do the same, as does the [service manual](https://www.gov.uk/service-manual). We talk, blog and write about what we're doing, and we are open about as much as we can.
```
Comment (rjw1) [inline playbook.md]: possibly link to our blog

**Comment by lozette** [comment]
I have resolved the conflicts that were in this PR - @harry-m can you check I've not accidentally removed any of your changes?

**Comment by lozette** [comment]
I have tried to resolve the conflicts - can someone please check I haven't accidentally removed anything? (@harry-m)

**Comment by harry-m** [comment]
LGTM 👍

**Pull request merged**